exporters/common/src/main/java/io/opentelemetry/exporter/internal/ExporterMetrics.java

```diff
@@ -58,21 +58,39 @@ public void addFailed(long value) {
     exported.add(value, failedAttrs);
   }
 
-  /** Create an instance for recording gRPC exporter metrics. */
+  /**
+   * Create an instance for recording exporter metrics under the meter "{@code
+   * io.opentelemetry.exporters.}" + {@code exporterName} + "-grpc".
```

Comment:
does this javadoc work???

Reply:
It certainly leaves room for improvement! Fixed.